### DISCUSSION TITLE
Live365 Ad Insertion

### DISCUSSION THREAD

**Discussion by wppronto** [AzuraCast/AzuraCast#7054] [Q&A]
Has anyone tried (and had success with) using Live365's ad triggers with Azuracast?

We've had mixed results...ads starting too early or the song after an ad starting too soon. It's odd the ad trigger files should theoretically work like anything else. But, I guess, Live365 is inserting its own overrides into the stream.

Any tips on smoothing this out will be appreciated.

I also love the Live365 now offers AAC, and bitrates above 128.

**Comment by Moonbase59** [comment]
I’ve not used Live365, but it seems they give you _templates_ of _30_, _60_, _90_ or _120 s_ in length, with a pre-defined Artist & Title, which they detect and override.

So the trick would be
- _not_ to suppress the metadata, and
- ensure no autocueing or fading happens (set cue/fade/overlap in AzuraCast correctly, I’d suggest 0.1s)

No guarantees, though, since I can’t test it. ;-)
